examples/pieces/some_constellations.scd

```diff
@@ -193,16 +193,16 @@ SynthDef(\strings, { arg out, freq=440, amp=0.1, gate=1, pan, freqLag=0.2;
 (
 Pdef(\vi,
 	Pbind(
-        \instrument, \strings,
-        \degree, Pseq([
-        			Pn(\rest, 2),
-        			Pshuf([0, 2, 3, 5], 10),
-        			Pseq([\rest], { 9.rand })
-        			], inf)
-        			+ Prand([0, 0, 0, [0, 3], [0, 1]], inf),
-        \dur, Pseq([1, 2, 0.3, 0.5, 0.5], inf) + (Prand([0.3, 0, 0.2], inf) * 0.1),
-        \detune, Pseg(Pwhite(-2, 2, inf), 1.3)
-       )
+		\instrument, \strings,
+		\degree, Pseq([
+			Pn(\rest, 2),
+			Pshuf([0, 2, 3, 5], 10),
+			Pseq([\rest], { 9.rand })
+		], inf)
+		+ Prand([0, 0, 0, [0, 3], [0, 1]], inf),
+		\dur, Pseq([1, 2, 0.3, 0.5, 0.5], inf) + (Prand([0.3, 0, 0.2], inf) * 0.1),
+		\detune, Pseg(Pwhite(-2, 2, inf), 1.3)
+	)
 ).play
 );
 )
```

Comment:
Elsewhere in this file, you'll find indentation that is idiomatic to the author, but departs from SC's convention, see [these lines](https://github.com/supercollider/supercollider/blob/762a6320698cfd73c362d727d662aae520259ac8/examples/pieces/some_constellations.scd#L181-L190) for example.
Do you want to catch this kind of thing in this reformatting?